pkg/controller/installation/core_controller_test.go

```diff
@@ -90,10 +90,6 @@ var _ = Describe("Testing core-controller installation", func() {
 				Expect(i.Spec.CalicoNetwork.IPPools).To(HaveLen(0))
 				return
 			}
-			if calicoNet.FlexVolInitContainerEnabled == nil {
-				t := true
-				Expect(i.Spec.CalicoNetwork.FlexVolInitContainerEnabled).To(Equal(&t))
```

Comment:
Rather than just remove this test, can we replace the Expect() with the corresponding check in the new code? Same for the other tests where it's been removed.

Reply:
Not sure what you mean here, there already are render tests that check that a FlexVolume container was created given the correct string in `node_test.go`

Reply:
But this isn't a render test, this is a defaulting test. The check here is meant to check that the `caliconet.Spec.FlexVolumePath` gets defaulted to the correct value if it isn't set.

Reply:
I added those to `defaults_test.go`, do you want me to duplicate them here?

Reply:
Nah that's ok. I was thrown off by them moving is all. So long as the tests exist I'm happy :+1: